Snail/Observable.swift

```diff
@@ -126,7 +126,7 @@ public class Observable<T>: ObservableType {
     public func filter(_ isIncluded: @escaping (T) -> Bool) -> Observable<T> {
         let filtered = Observable<T>()
 
-        subscribe(
+        _ = subscribe(
```

Comment:
I think we should add this to the subscriber's array right? This way we remove this subscription when `deinit`